buildlib/pr/main.yml

```diff
@@ -8,6 +8,9 @@ resources:
     - container: fedora34
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/fedora34:2
       options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools
+    - container: conda
+      image: rdmz-harbor.rdmz.labs.mlnx/ucx/conda
```

Comment:
please set correct tag for docker

Reply:
What's the correct tag here? I don't set it anywhere and believe this is done when building the Docker container, I don't know where that's handled but it doesn't seem to be from the UCX code repo.

Reply:
you should build and push docker to hardor , when you push docker you set tag for your image

Reply:
I don't think I have access to that network. Could someone from your team please do it?

Reply:
@avildema can you pls assist with pushing the docker image to harbor?

Reply:
Actually, it seems that network is indeed reachable from part of the network I'm connected to. I don't have access to the machine nevertheless, so if you prefer we can sync on how I can get access to it so I can build the image.